src/lib/types/apiRequests.ts

```diff
@@ -416,6 +417,14 @@ export interface CreateDiscordEventRequest {
   image?: string;
 }
 
+export interface PatchDiscordEventRequest extends CreateDiscordEventRequest {
+  eventID: string;
+}
+
+export interface DeleteDiscordEventRequest {
+  eventID: string;
```

Comment:
I think we can use type `UUID` here since it's defined up in the file that `type UUID = string`